src/wolfgame.py

```diff
@@ -6861,6 +6887,7 @@ def start(cli, nick, chan, forced = False, restart = ""):
         cgamemode(cli, restart)
         var.GAME_ID = time.time() # restart reaper timer
 
+    var.START_VOTES = set()
```

Comment:
Move both this and line 84 into the reset() definition, so that it's properly reset if e.g. game times out or other stuff.